bin/diagnose-voice.sh

```
@@ -6,15 +6,26 @@
 #   - The pipecat-bridge container started with DEBUG=1 (gates the synthetic endpoint)
 #   - $BRIDGE_JWT_SECRET exported (matches the value in .env)
 #   - tests/fixtures/voice-intake-sample.wav (commit-tracked or generated locally)
+#
+# Optional:
+#   - $COMPOSE_PROJECT_NAME (default: parent dir name) — must match the project
+#     used at `docker compose ... up`. Without this the ps lookups silently
+#     report "not running" even when services are up under a non-default project.
+#   - $BRIDGE_HOST_PORT (default: 8400) — host port the bridge is published on.
 
 set -uo pipefail
 cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)